packages/vue3-vite/src/generators/vue3-vite/files/jest.config.ts.template

```diff
@@ -1,19 +1,19 @@
 module.exports = {
   displayName: '<%= projectName %>',
-  preset: '<%= offsetFromRoot %>jest.preset.js',
   globals: {
     'vue-jest': {
       tsConfig: './<%= projectRoot %>/tsconfig.spec.json',
     },
   },
   transform: {
     '^.+\\.vue$': '@vue/vue3-jest',
-    '^.+\\.[jt]sx?$': 'ts-jest',
+    "^.+\\.jsx$": 'babel-jest',
```

Comment:
Added `babel-jest` transformer for JS support. Since the lib generator comes with a component that has `<script setup>` (e.g. JS), the test won't work unless we use the correct transformer.

I think it should be possible for ts-jest to handle both situations, but couldn't figure it out.